tests/gtest/avifdecodetest.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include <array>
+#include <tuple>
```

Comment:
Nit: Delete these two. Include <iostream> and <string>.

Reply:
Done.